src/scenarios/blazor/pre.py

```diff
@@ -9,7 +9,9 @@
 
 # For blazor3.2 the linker argument '--dump-dependencies' should be added statically to enable linker dump
 # For blazor5.0 the linker argument can be added to the command line as an msbuild property
+
 subprocess.run(["dotnet", "workload", "uninstall", "microsoft-net-sdk-blazorwebassembly-aot"])
```

Comment:
Should "microsoft-net-sdk-blazorwebassembly-aot" uninstall be removed.

Reply:
I want to keep it around to make sure that package gets removed from all of our machines. We can remove it once it has run everywhere.